### ISSUE TITLE
pc softmax and LADE in other domain

### ISSUE THREAD

**Issue by aiexperience** [hyperconnect/LADE#15]
Currently, I'm conducting an experiment to solve the imbalance problem in the 1d signal domain. I have a question about your research, so I opened the question because I wanted to ask for an opinion.

My experiment is a task that classifies normal and various failure types in the imbalanced dataset. Experiments are underway with pc softmax and LADE. Most of the data is normal and others are failure types (unbalance, misalignment ...). In many experiments, the LADE performs better than other loss types in this domain, too! (e.g. focal loss, causal norm ..)

And here are two things that I am curious about.

The first is the difference in accuracy between pc softmax and lade. In the image domain (in your research), the accuracy of the two methods was similar, whereas, in the 1d signal currently being experimented, pc softmax showed lower results than the LADE. (about 10% accuracy gap) 
Due to the characteristics of the data, I am proceeding without performing normalization, and I'm curious about whether this 'normalization' can affect the accuracy of the pc softmax. Since the model output logit is directly modified, I think it will be affected by using values that are not normalized, and I would like to hear your opinion on this.

The second is the effect of LADER in the LADE loss (LADE-CE + LADER). LADE loss consists of two losses. Personally, I was curious about LADER loss. So I experimented using only LADER loss (without cross-entropy), the accuracy was similar to that of the LADE. I would appreciate it if you could give me an opinion on the reason for this high accuracy even though cross-entropy has not been applied.

Thank you!

**Comment by juice500ml** [comment]
1. On normalization
    - Note that I'm not sure what kind of normalization is conducted, so let me be a bit less confident on answering this.
    - But still, I'm a bit surprised that there are huge accuracy gap. Maybe the regularizing nature of LADE is helping the performance. If you see the Figure 3 in our paper, you can see that more regularization yields less variance across different class logit values. If I were you, I'll try lots of different alpha (eq. 20) and lambda (eq. 10) to fully understand what's going on.

2. Effect of LADER
    - Yes, using only LADER loss can yield feasible results. However, in image datasets, adding CE loss somewhat helped, especially that LADER doesn't yield normalized probabilities.
    - CE has the assumption of single label, where LADER doesn't. That was why we thought adding CE helped, especially for image datasets such as CIFAR. However, without those kind of assumptions, there are no reason for LADER to underperform, especially that LADER and CE loss has a bit of a similarity between the two.

We welcome any kind of additional discussions! Thank you for asking in-depth questions on our paper. We could co-work if you need us to be involved in solving your problem. [Email me](mailto:juice500ml@gmail.com) if you need to.
